conf/modules/tlsf.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="tlsf">
+  <doc>
+    <description>
+      Dynamic memory allocator
+      Can be used as a replacement of malloc with a O(1) allocation time
```

Comment:
Then it would be good to describe in the readme in sw/ext/tlsf which version it was based on and what changes were made and why...